include/envoy/stats/store.h

```diff
@@ -43,6 +43,11 @@ class Store : public Scope {
    * @return a list of all known histograms.
    */
   virtual std::vector<ParentHistogramSharedPtr> histograms() const PURE;
+
+  /**
+   * @return whether any known counter exists with this name.
+   */
+  virtual bool counterExists(const std::string& counter_name) PURE;
```

Comment:
cc @ahedberg for intersection with https://github.com/envoyproxy/envoy/pull/6712. Can we intersect with the APIs being added there so we don't duplicate?

Reply:
This one is test only, so it will definitely be best to just get rid of counterExists, and use those ones in here. That will be a very easy switch; left a todo.